server/game/cards/01-Core/RitualOfBalance.js

```diff
@@ -6,9 +6,13 @@ class RitualOfBalance extends Card {
             condition: context => context.player.opponent && context.player.opponent.amber >= 6,
             gameAction: ability.actions.steal()
         });
+        this.action({
```

Comment:
I don't think this is the correct way to fix this bug.  The problem is that the game won't allow the artifact to be used when the action condition isn't met, whereas it should allow the ability to be used, and just not permit the game action to resolve

Reply:
I'm sure there's a more clever and cleaner way to do it... Not sure how to put a condition inside the gameAction though, as there are no other cards that work in a similar way.
On top of my head, stuff like Anomaly Exploiter with no damaged creatures should exhaust just fine as the condition is on the selection...

Reply:
Hmmm, I'll take a look